README.md

```diff
@@ -69,11 +69,20 @@ Example:
 ```js
 const fs = require('fs-extra')
 
+// Async with promises:
+fs.copy('/tmp/myfile', '/tmp/mynewfile')
+  .then(() => console.log("success!"))
```

Comment:
Yeah, we should fix that; thanks for the catch. I also removed an old semicolon in these docs. :wink: 

Someone ought to make a tool to run eslint on code blocks in markdown.